### PR TITLE
[L2B-7355] Fix tooltip issue

### PR DESCRIPTION
**Pull request by maciekop-l2b**

### CODE REVIEW

File: packages/uops-dashboard/next.config.mjs [hunk 1]
```diff
@@ -3,4 +3,5 @@ const nextConfig = {
   reactStrictMode: true,
 }
 
+// biome-ignore lint/style/noDefaultExport: <explanation>
```
Comment (torztomasz) [inline packages/uops-dashboard/next.config.mjs:6]: image

File: packages/uops-dashboard/package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@types/uuid": "^10.0.0",
     "ethers": "^5.7.2",
-    "flowbite-react": "^0.10.1",
+    "flowbite-react": "0.10.1",
```
Comment (torztomasz) [inline packages/uops-dashboard/package.json:18]: why do we need that?

File: packages/uops-dashboard/package.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
     "eslint": "^8",
     "eslint-config-next": "14.2.6",
     "postcss": "^8",
-    "tailwindcss": "^3.4.1",
+    "tailwindcss": "3.4.1",
```
Comment (torztomasz) [inline packages/uops-dashboard/package.json:33]: why locking the version?

File: packages/uops-dashboard/postcss.config.mjs [hunk 4]
```diff
@@ -5,4 +5,5 @@ const config = {
   },
 }
 
+// biome-ignore lint/style/noDefaultExport: <explanation>
```
Comment (torztomasz) [inline packages/uops-dashboard/postcss.config.mjs:8]: image

File: packages/uops-dashboard/tailwind.config.ts [hunk 5]
```diff
@@ -25,4 +25,5 @@ const config: Config = {
     },
   ],
 }
+// biome-ignore lint/style/noDefaultExport: <explanation>
```
Comment (torztomasz) [inline packages/uops-dashboard/tailwind.config.ts:28]: image

**Review by torztomasz — CHANGES_REQUESTED** [review]

File: packages/uops-dashboard/package.json (hunk 2 above)
Comment (maciekop-l2b) [reply packages/uops-dashboard/package.json:18]: Removed

File: packages/uops-dashboard/package.json (hunk 3 above)
Comment (maciekop-l2b) [reply packages/uops-dashboard/package.json:33]: unlocked

File: packages/uops-dashboard/postcss.config.mjs (hunk 4 above)
Comment (maciekop-l2b) [reply packages/uops-dashboard/postcss.config.mjs:8]: Fixed, was not referencing root biome config file

File: packages/uops-dashboard/tailwind.config.ts (hunk 5 above)
Comment (maciekop-l2b) [reply packages/uops-dashboard/tailwind.config.ts:28]: Fixed, was not referencing root biome config file

File: packages/uops-dashboard/next.config.mjs (hunk 1 above)
Comment (maciekop-l2b) [reply packages/uops-dashboard/next.config.mjs:6]: Fixed, was not referencing root biome config file

**Pull request merged**